app/forms/hyrax/hyrax_form.rb

```diff
@@ -46,7 +46,7 @@ def rights_statement
       if Array(self["rights_statement"]).first.blank?
         "http://rightsstatements.org/vocab/NKC/1.0/"
       else
-        self["rights_statement"]
+        Array(self["rights_statement"]).first
```

Comment:
It looks like self['rights_statement'] is getting initialized to an empty string, but I can make the conditional do a bunch less casting I think.